lib/service/service.go

```diff
@@ -3837,6 +3841,11 @@ func (process *TeleportProcess) WaitWithContext(ctx context.Context) {
 // StartShutdown launches non-blocking graceful shutdown process that signals
 // completion, returns context that will be closed once the shutdown is done
 func (process *TeleportProcess) StartShutdown(ctx context.Context) context.Context {
+	// by the time we get here we've already extracted the parent pipe, which is
+	// the only potential imported file descriptor that's not a listening socket
+	warnOnErr(process.closeImportedDescriptors(""), process.log)
```

Comment:
Does `closeImportedDescriptors` with empty "" type make any effect ? 

It looks that the parent process [exports](https://github.com/gravitational/teleport/blob/9d91466a0ecbe5f22fc85a41011dd820cd3f0594/lib/service/signals.go#L421) only listener + signalPipeName so all imported descriptors should have `fd.Type` set.

Reply:
The argument to `closeImportedDescriptors` is a prefix, it'll close everything.

Reply:
Right 👍 
nit: Could you extent the comment ?